### PR TITLE
NET-4952 Add docs for export command

### PR DESCRIPTION
**Pull request by nathancoleman**
### Description

Consul 1.16 introduced the `services export` command in the Consul CLI via #14547 / #15654 , and I intended to add docs at the time but forgot. These are the docs.

### Testing & Reproduction steps

Verify docs match usage introduced in #15654.

### Links
[Vercel deploy](https://consul-git-export-docs-hashicorp.vercel.app/consul/commands/services/export)

### PR Checklist

* [ ] updated test coverage
* [x] external facing docs updated
* [x] appropriate backport labels added
* [x] not a security concern

### CODE REVIEW

**Review by 20sr20 — APPROVED** [review]
LGTM! Thanks for adding the doc.

**Review by david-yu — APPROVED** [review]
Thanks for doing this, the docs team probably needs to review formatting but overall looks good to me.

File: website/content/commands/services/export.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+@include 'http_api_namespace_options.mdx'
+
+#### API Options
+
+@include 'http_api_options_client.mdx'
+
+## Examples
+
+The example below shows using the `export` command to export the `web` service to a cluster
+peer named `dc2`.
```
[the remark this answers is not in the document]
Comment (nathancoleman) [reply website/content/commands/services/export.mdx:54]: Much better, thanks!

File: website/content/commands/services/export.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+```shell-session hideClipboard
+$ consul services export -name=web -consumer-peers=dc2
+```
+
+The example below shows using the `export` command to export the `web` service located in the
+namespace `ns1` and the admin partition `part1` to other admin partitions named `part2` and `part3`.
```
[the remark this answers is not in the document]
Comment (nathancoleman) [reply website/content/commands/services/export.mdx:61]: alpha, beta and delta seem reasonable to me. I agree that `partition-1` isn't ideal in an example like this

**Pull request merged**